### ISSUE TITLE
Reduce Spacing on Table headers

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/platform-ui#643]
![image](https://user-images.githubusercontent.com/15824252/65353631-9f549a80-dbbb-11e9-87e3-82a095f25c72.png)

**Comment by rosibaj** [comment]
@kcullion Can we determine some exact measurements here?

**Comment by kcullion** [comment]
@rosibaj here you go....
![Screen Shot 2019-09-23 at 9 37 03 AM](https://user-images.githubusercontent.com/18424138/65430332-be367500-dde5-11e9-8290-6eb062463c0b.png)

**Comment by kcullion** [comment]
Actually @rosibaj as per Jon's point in slack, I was going to suggest saying "Row #' in the first column header because it will be reflecting the same row in the excel sheet - so if reordering by another column, the row numbers stay with the donors. So in that case the first width will be wider...

**Comment by rosibaj** [comment]
@kcullion maybe just `Row`, no `#` needed?

**Comment by kcullion** [comment]
yeah @rosibaj maybe we can just leave it as #...it's prob intuitive enough and then it doesn't have to be too wide

**Comment by rosibaj** [comment]
@ciaranschutte this ticket has no PR attached?  Has it been merged already?
